.gitattributes

```diff
@@ -0,0 +1,5 @@
+Resources/Private/.eslintignore export-ignore
+Resources/Private/.eslintrc export-ignore
+Resources/Private/.prettierrc.js export-ignore
+Resources/Private/package.json export-ignore
+Resources/Private/stylelint.config.js export-ignore
```

Comment:
Please let's have a LF after the last line so adding a new line does not modify the not-the-last-anymore line. :-)